network/p2p/sporking_test.go

```diff
@@ -36,42 +37,38 @@ func TestSporkingTestSuite(t *testing.T) {
 // TestCrosstalkPreventionOnNetworkKeyChange tests that a node from the old chain cannot talk to a node in the new chain
 // if it's network key is updated while the libp2p protocol ID remains the same
 func (suite *SporkingTestSuite) TestCrosstalkPreventionOnNetworkKeyChange() {
-
 	// create and start node 1 on localhost and random port
 	node1key := generateNetworkingKey(suite.T())
-	node1, address1 := suite.CreateNode("node1", node1key, "0.0.0.0", "0", rootBlockID, nil, false)
+	node1, id1 := suite.NodeFixture(node1key, rootBlockID, nil, false, defaultAddress)
 	defer suite.StopNode(node1)
-	suite.T().Logf(" %s node started on %s:%s", node1.name, address1.IP, address1.Port)
-	suite.T().Logf("libp2p ID for %s: %s", node1.name, node1.host.ID())
+	suite.T().Logf(" %s node started on %s", id1.NodeID.String(), id1.Address)
+	suite.T().Logf("libp2p ID for %s: %s", node1.id.String(), node1.host.ID())
 
 	// create and start node 2 on localhost and random port
 	node2key := generateNetworkingKey(suite.T())
-	node2, address2 := suite.CreateNode("node2", node2key, "0.0.0.0", "0", rootBlockID, nil, false)
+	node2, id2 := suite.NodeFixture(node2key, rootBlockID, nil, false, defaultAddress)
 
 	// create stream from node 1 to node 2
-	testOneToOneMessagingSucceeds(suite.T(), node1, address2)
+	testOneToOneMessagingSucceeds(suite.T(), node1, id2)
 
 	// Simulate a hard-spoon: node1 is on the old chain, but node2 is moved from the old chain to the new chain
 
 	// stop node 2 and start it again with a different networking key but on the same IP and port
 	suite.StopNode(node2)
 
-	// generate a new key
+	// start node2 with the same name, ip a`nd port but with the new key
 	node2keyNew := generateNetworkingKey(suite.T())
 	assert.False(suite.T(), node2key.Equals(node2keyNew))
-
-	// start node2 with the same name, ip and port but with the new key
```

Comment:
thanks for catching the wrong comment earlier

Reply:
🙏